public/js/app/pages/__tests__/__snapshots__/CourseMemo.test.js.snap

```diff
@@ -49,7 +49,7 @@ exports[`Page CourseMemo renders a side menu and a page structure 1`] = `
     </ul>
   </section>
   <div
-    class="p-print-4 row"
+    class="archivedMemoPage row"
```

Comment:
Depending on the comment from `CourseMemo.jsx`, this might also have to be changed? Should it not be both classNames? `p-print-4` and `archivedMemoPage` ?